holidays/countries/sierra_leone.py

```diff
@@ -0,0 +1,132 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.groups import (
+    ChristianHolidays,
+    InternationalHolidays,
+    IslamicHolidays,
+    StaticHolidays,
+)
+from holidays.observed_holiday_base import (
+    ObservedHolidayBase,
+    SAT_SUN_TO_NEXT_WORKDAY,
+)
+
+
+class SierraLeone(
+    ObservedHolidayBase,
+    ChristianHolidays,
+    InternationalHolidays,
+    IslamicHolidays,
+    StaticHolidays,
+):
+    """Sierra Leone holidays.
+
+    References:
+        * https://www.officeholidays.com/countries/sierra-leone
+        * https://en.wikipedia.org/wiki/Public_holidays_in_Sierra_Leone
+        * https://www.timeanddate.com/holidays/sierra-leone/
```

Comment:
```suggestion
        * <https://en.wikipedia.org/wiki/Public_holidays_in_Sierra_Leone>
        * <https://www.officeholidays.com/countries/sierra-leone>
        * <https://www.timeanddate.com/holidays/sierra-leone/>
```
It would be good to have some official sources.

Reply:
❗